### DISCUSSION TITLE
! КОНЦЕПТ ! "Исключение игроков"

### DISCUSSION THREAD

**Discussion by Vladislav894** [Sokol-Games-GmbH/PixelCombats.PublicApi#250] [Ideas]
Создал концепт по исключению игроков
1.Как оно работает?
Исключение игроков работает очень просто: Нам нужно нажать на меню и выбрать кнопку "kicked". Затем появится, что-то вроде лидерборда при нажатии на таймер, там отображаются обычные игроки и хост (хост помечен буквой H), чтобы кикнуть нужно нажать на "kick", после чего появится окно "Исключить игрока?". Нажав на "ок" в игре появится сообщение "игрок исключен!" а у того, кого исключили на весь экран появится красным сообщением "YOU KICKED".
2. Другое
Огромный плюс этой функции в том, что можно исключать неадекватных игроков, а так-же читеров, так-же можно избавляться от гриферов
Так-же эту функцию можно будет включить/выключить (по умолчанию будет включена) в меню создания мира
Эта функция будет работать в мире и редакторе, так-же можно будет делать в кастомных режимах игроков с помощью кодов
ТАК-ЖЕ ПРИДОМТАВЛЕНЫ СКРИНЫ КОНЦЕПТА!
![Без названия74_20240225173309](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/148636358/f8ec7bfb-e5cc-4174-85a3-3ad8479dcc48)
![Без названия72_20240225172800](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/148636358/c9af1a46-d872-45a0-9105-a04a3f757d7b)
![Без названия71_20240225172701](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/148636358/5754eed7-a681-43b5-8e7a-784918e761e3)
![Без названия71_20240225172239](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/148636358/84a57ec3-c5dc-4961-9401-4c56f85f9141)
![Без названия70_20240225171902](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/148636358/53db5080-3d9f-4670-a8ca-8a83a0e39bd1)

**Comment by SobakaBarabaka74** [comment]
Понимаю что вам хочется мира и спокойствия, но функция слишком не безопасна

**Comment by kkohno** [comment]
согласен с мнением выше - будут выкидывать просто так

**Reply by KELHUN** [reply]
Ну можно же сделать чтобы игрока кикало только при согласии больше половины сервера, то есть игрок выбирает игрока и на экране где нибудь в углу будет выбор с таймером изгнать или оставить.

**Reply by SobakaBarabaka74** [reply]
Тоже плохо, вдруг игрокам игрок тупо не нравится

**Reply by KELHUN** [reply]
Ну ведь если игрок не понравился половине сервера,значит на то есть причина,например Грифф,оскорбление и т.д. +я думаю что это не особо повлияет на таких людей,они смогут просто перезайти.

**Reply by SobakaBarabaka74** [reply]
Так смысл тогда вообще делать это? если игрок просто перезайдет
